.JuliaFormatter.toml

```diff
@@ -1,4 +1,5 @@
 style = "sciml"
 format_markdown = true
 annotate_untyped_fields_with_any = false
-format_docstrings = true
\ No newline at end of file
+format_docstrings = true
+join_lines_based_on_source = false
```

Comment:
@thazhemadam are we able to standardize this in the .github?

Reply:
`annotate_untyped_fields_with_any = false` is a better case only if ConcreteStructs are being used

Reply:
Though `join_lines_based_on_source = false` makes the formatting more standard across different authors